packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -176,141 +172,4 @@ export interface IContextualMenuProps extends React.Props<ContextualMenu> {
    */
   calloutProps?: ICalloutProps;
 
-}
```

Comment:
Good call. Added to the doc page. Also remembered to deprecate itemType (as that value will conflict with using it in button scenerio).